spec/spec_helper.rb

```diff
@@ -1,10 +1,14 @@
 require 'rubygems'
 require 'bundler/setup'
+require 'rspec/collection_matchers'
 
 $LOAD_PATH.unshift(File.join(File.dirname(__FILE__), '..', 'lib'))
 $LOAD_PATH.unshift(File.dirname(__FILE__))
 
 require 'cloudsearchable'
 
-RSpec.configure do |config|
+RSpec.configure do |rspec|
+  rspec.expect_with :rspec do |c|
+    c.warn_about_potential_false_positives = false

```

Comment:
But we're turning off all warnings of this type, when we want to ignore this warning in a specific case. The risk is that later we'll add/change a test where the warning is meaningful, but we won't see it because they're all disabled. I'd rather live with a spurious warning than risk missing a real one.